loops-arrays-objects.js

```diff
@@ -9,15 +9,16 @@ const scores = [
     {name: 'Rianne', score: 66, grade: null}
 ];
 
+
 // Verwachtte uitkomsten:
 // 83
 // 77
 // 92
 // 66
 // ==========================================
-
-
-
+for (let i = 0; i < scores.length; i++) {
+    console.log(scores[i]["score"]);
```

Comment:
Het gebruik van console.log om scores te printen is eenvoudig en effectief. De for-lus is begrijpelijk en duidelijk.